src/menu_events.cpp

```diff
@@ -454,6 +454,29 @@ void menu_handler::show_enemy_moves(bool ignore_units, int side_num)
 			gui_->highlight_another_reach(path);
 		}
 	}
+	
+	//   Fetch cursor position, in order to later find a possible
+	// unit under it.
+	int x = -1;
```

Comment:
`x` and `y` are no longer used, so please remove them, the comment, and the call to `SDL_GetMouseState()`.